src/snowflake/cli/_plugins/workspace/commands.py

```diff
@@ -57,4 +58,5 @@ def bundle(
         project_root=cli_context.project_root,
     )
 
-    ws.perform_action(entity_id, EntityActions.BUNDLE)
+    bundle_map: BundleContext = ws.perform_action(entity_id, EntityActions.BUNDLE)
```

Comment:
is this right? the bundle action seems to return a bundle map, not a context instance